go.mod

```diff
@@ -284,3 +284,5 @@ require (
 	lukechampine.com/blake3 v1.2.1 // indirect
 	nhooyr.io/websocket v1.8.7 // indirect
 )
+
+replace github.com/onflow/flow-go/crypto => ./crypto
```

Comment:
This is needed to integrate the Flow crypto build changes into flow-go. 
In particular CI workflow wouldn't work correctly if flow-go is till using the Relic-based crypto while flow-go/crypto is not. 

The plan to remove this line is to make these following steps:
 - merge this current PR
 - tag the new crypto lib
 - remove this line and update go.mod to use the new tag